platforms/jvm/plugins-groovy/src/main/java/org/gradle/api/plugins/GroovyPlugin.java

```diff
@@ -52,4 +61,16 @@ private void configureGroovydoc(final Project project) {
             groovyDoc.setSource(groovySourceSet);
         });
     }
+
+    private void configureCompileDefaults(final Project project) {
```

Comment:
Why is this in `groovy` and not `groovy-base`?

This seems like a general "configure compile task that is created" instead of "configure the component specifically created by this plugin"